hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/AtomicCryptoTransferHTSSuite.java

```diff
@@ -1437,4 +1437,42 @@ final Stream<DynamicTest> blockCryptoTransferForPermittedDelegates() {
                                         .contractCallResult(
                                                 htsPrecompileResult().withStatus(SPENDER_DOES_NOT_HAVE_ALLOWANCE)))));
     }
+
+    @HapiTest
+    final Stream<DynamicTest> nullContractAdminKeyTransfer() {
+        final var nullAdminKeyXferTxn = "nullAdminKeyXferTxn";
+        return hapiTest(
+                cryptoCreate(RECEIVER),
+                uploadInitCode(CONTRACT),
+                contractCreate(CONTRACT).omitAdminKey(),
+                cryptoTransfer(tinyBarsFromTo(GENESIS, CONTRACT, ONE_HUNDRED_HBARS)),
+                withOpContext((spec, opLog) -> {
+                    final var receiver = spec.registry().getAccountID(RECEIVER);
+                    final var contract = spec.registry().getAccountID(CONTRACT);
+                    final var amountToBeSent = 50 * ONE_HBAR;
+                    allRunFor(
+                            spec,
+                            contractCall(
+                                            CONTRACT,
+                                            TRANSFER_MULTIPLE_TOKENS,
+                                            transferList()
+                                                    .withAccountAmounts(
+                                                            accountAmount(contract, -amountToBeSent, false),
+                                                            accountAmount(receiver, amountToBeSent, false))
+                                                    .build(),
+                                            EMPTY_TUPLE_ARRAY)
```

Comment:
I see this pattern in several tests in this file - but I don't get why there's both a `transferList()...build()` argument and an `EMPTY_TUPLE_ARRAY` argument.  The method in the contract takes a single argument of type `TokenTransferList[]`.  How do these two arguments "coaelsce" or something into one `TokenTransferList[]`?